Sectors/Lines - LACC.txt

```diff
@@ -378,13 +378,24 @@ COORD:N052.28.42.000:E002.34.14.000
 COORD:N052.28.19.000:E002.46.44.000
 
 SECTORLINE:LCE1
-DISPLAY:London AC Daventry:London AC Daventry:London AC Clacton
-DISPLAY:London AC Clacton:London AC Clacton:London AC Daventry
-DISPLAY:London TC DAGGA:London TC DAGGA:London AC Daventry
-DISPLAY:London S10:London AC Daventry:London AC Clacton
-DISPLAY:London S11:London AC Daventry:London AC Clacton
-DISPLAY:London TC CPT:London AC Daventry:London AC Clacton
-DISPLAY:London AC Dover:London AC Daventry:London AC Clacton
+DISPLAY:London TC DAGGA:London TC DAGGA:London TC WELIN
+DISPLAY:London TC WELIN:London TC WELIN:London TC DAGGA
+DISPLAY:London AC Clacton:London AC Clacton:London TC WELIN
+DISPLAY:London TC WELIN:London TC WELIN:London AC CLACTON
+DISPLAY:London TC DAGGA:London TC DAGGA:London S28
+DISPLAY:London S28:London S28:London TC DAGGA
```

Comment:
If I'm being picky, introducing the Daventry resectorisation shouldn't be in this issue otherwise they become co-dependent! Will have to hold off on merging this issue now until the other is reviewed